src/main/groovy/com/calgaryscientific/gradle/GenerateToUploadTask.groovy

```diff
@@ -31,7 +31,8 @@ class GenerateToUploadTask extends VeracodeTask {
 
     GenerateToUploadTask() {
         description = "Grabs all jar files from the dir provided and filter it into a the 'build/to-upload' folder"
-        requiredArguments << 'dir' << "force${VeracodeTask.OPTIONAL}"
+        requiredArguments << 'dir'
```

Comment:
what's dir and what's force for?

Reply:
They are part of the existing code. This PR is just making the optional properties definition less hacky.

Reply:
The current state of this code is that it copies *.jar files in the project into the to-upload dir and uploads that.
In my dev branch I have a config option that reads the files from a file list instead. I'll submit a PR for that later.